wizard/WizardDaemonSettings.qml

```diff
@@ -0,0 +1,198 @@
+            Layout.fillWidth: true
+            Layout.topMargin: 30 * scaleRatio
+            Layout.bottomMargin: 30 * scaleRatio
+            font.family: "Arial"
+            font.pixelSize: 18 * scaleRatio
+            wrapMode: Text.Wrap
+            //renderType: Text.NativeRendering
+            color: "#4A4646"
+            textFormat: Text.RichText
+//            horizontalAlignment: Text.AlignHCenter
+            text: qsTr("To be able to communicate with the Monero network your wallet needs to be connected to a Monero node. For best privacy it's recommended to run your own node.
+                        <br><br>
+                        If you don't have the option to run an own node there's an option to connect to a remote node.")
+                    + translationManager.emptyString
+        }
+    }
+
+    ColumnLayout {
+
+        RowLayout {
+            CheckBox {
+                id: localNode
+                text: qsTr("Start a node automatically in background (recommended)") + translationManager.emptyString
+                background: "#FFFFFF"
+                fontColor: "#4A4646"
+                fontSize: 16 * scaleRatio
+                checkedIcon: "../images/checkedVioletIcon.png"
+                uncheckedIcon: "../images/uncheckedIcon.png"
+                checked: appWindow.persistentSettings.startLocalNode && !isAndroid && !isIOS
+                visible: !isAndroid && !isIOS
+            }
+        }
+
+        ColumnLayout {
+            visible: localNode.checked
+            id: blockchainFolderRow
+            Label {
+                Layout.fillWidth: true
+                Layout.topMargin: 20 * scaleRatio
+                fontSize: 14 * scaleRatio
+                text: qsTr("Blockchain location") + translationManager.emptyString
+            }
+            LineEdit {
+                id: blockchainFolder
+                Layout.preferredWidth:  200 * scaleRatio
+                Layout.fillWidth: true
+                text: persistentSettings.blockchainDataDir
+                placeholderText: qsTr("(optional)") + translationManager.emptyString
+
+                MouseArea {
+                    anchors.fill: parent
+                    onClicked: {
+                        mouse.accepted = false
+                        if(persistentSettings.blockchainDataDir != "")
+                            blockchainFileDialog.folder = "file://" + persistentSettings.blockchainDataDir
+                        blockchainFileDialog.open()
+                        blockchainFolder.focus = true
+                    }
+                }
+
+            }
+        }
+
+        RowLayout {
+            CheckBox {
+                id: remoteNode
+                text: (localNode.checked) ? qsTr("Connect to a 3rd party remote node until my own node has finished syncing") + translationManager.emptyString
+                                          : qsTr("Connect to a 3rd party remote node") + translationManager.emptyString
```

Comment:
I'd remove 3rd party since it doesn't have to be. Doesn't really matter though.

Reply:
removed